lib/src/screens/InitialScreen.dart

```diff
@@ -33,7 +33,7 @@ class _MyAppState extends State<MyApp> with SingleTickerProviderStateMixin {
   @override
   Widget build(BuildContext context) {
 
-    const primaryColorBar = const Color(0xFF717DBC);
+    const primaryColorBar = const Color(0xff6055cd);
```

Comment:
Not a request but a comment. It would be cool to create a color library. I could be in the "helpers" folder.